compiler/src/dmd/expressionsem.d

```diff
@@ -5282,10 +5283,10 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
                     tthis.modToBuffer(&buf);
 
                 //printf("tf = %s, args = %s\n", tf.deco, (*arguments)[0].type.deco);
-                .error(exp.loc, "%s `%s%s` is not callable using argument types `%s`",
+                .error(argLoc, "%s `%s%s` is not callable using argument types `%s`",
```

Comment:
I think this error should use `exp.loc` as it doesn't mention a particular argument - the supplemental one can still use `argLoc` (if it is set).
Ditto for line 5365.